.travis.yml

```diff
@@ -1,9 +1,12 @@
 language: go
 
 go:
-  - 1.3
-  - 1.4
   - release
```

Comment:
I'd explicitly set 1.9.x (yes, we'll need to update it but it forces us to re-run CI when doing so).